src/pages/Training/Calendar.tsx

```diff
@@ -0,0 +1,219 @@
+import { useState, useEffect, FC } from 'react';
```

Comment:
1. Modal do dodawania nowego eventu powinien byc oddzielnym komponentem. 
2. Dodawanie eventu to formularz- nie powinnismy uzywac state'ow tylko zrobic normalnie formularz z uzyciem react-hook-form. Daje to nam walidacje i pelna kontrole nad formularzem.
3. Na razie na onSubmit zrob console.log wartosci
------------------------------------------------------------
To jest kolejny task (nowa PRka): 
1. Na `onSubmit` powinienes wysylac nowy event do backendu i dodawac go do kolekcji events. 
 - zrob nowa kolekcje
 - zrob nowy endpoint
2. Zrob nowy endpoint do pobierania eventow
- powinien on przyjmowac jako request.params `startDate` i `endDate`. 
- na poziomie controllera musisz obsluzyc to tak, zeby pobrac tylko eventy dla wymienionych dat [operatory $lte i $gre](https://stackoverflow.com/questions/2943222/find-objects-between-two-dates-mongodb)
- po pobraniu musisz naniesc eventy na kalendarz (to bedzie ciezka spraw, moze sie pojawic rozjazd w datach i ich formatach, bedzie trzeba sie zastanowic co i jak ugryzc, mam na to pomysl, wiec ogarniemy to na lekcji najwyzej).
--------------------------------------------------------------------------------------------------------
Odpowiedzi na Twoje pytania: 
1. Eventy jak najbardziej powinny miec swoja kolekcje w bazie, nie widze innej opcji. Ja nazwalbym to `calendarEvents`, poniewaz samo `events` jest powiazane z czyms innym w nomenklaturze technicznej
2. Nie ma potrzeby tworzenia tutaj contextu. `AddEventModal` sluzy dodawaniu eventow, a `Calendar` ich pobieraniu, wystarczy stworzyc `useEffect`, ktory bedzie sie odpalal na podstawie zmiany np `openModal`
3. Dodawanie i usuwanie nie ma nic do rzeczy. Jeszcze raz przestudiuj sobie czym jest context i po co go uzywamy, bo skoro zastanawiasz sie nad tym czy go tworzyc w danej sytuacji to znaczy, ze nie do konca rozumiesz do czego on sluzy. Co z tego, ze bedziemy je edytowac i usuwac skoro bedzie sie to odbywac w innych komponentach (zapewne)? Czy potrzebujemy utrzymywania state'a w przypadku event'ow w kalendarzu? Czy kalendarz jest czyms dynamicznym? Czy moze synchronicznie przetwarza operacje na eventach i po prostu pobiera zmiany? Powinienes na tym poziomie juz widziec po co jest context i do czego go uzywamy, to nie jest zbior reuzywalnych funkcji czy wartosci.